browser/ui/views/frame/vertical_tabs/vertical_tab_strip_region_view.cc

```diff
@@ -274,6 +274,13 @@ class VerticalTabNewTabButton : public BraveNewTabButton {
 
  private:
   void UpdateColors() override {
+    auto* widget = GetWidget();
```

Comment:
Base class also has early return condition.
This crash caused from accessing null color_provider?
```
void TabStripControlButton::UpdateColors() {
  const auto* const color_provider = GetColorProvider();
  if (!color_provider) {
    return;
  }
  ...
```